### PR TITLE
Send kenzu skills to client

### PR DESCRIPTION
**Pull request by Nico-Sanchez**
> [!WARNING]
> Merge with https://github.com/lambdaclass/champions_of_mirra/pull/2014
## Motivation

We need to send the combo sequence to the client to change its animation.

## Summary of changes

- [Add new current_basic_animation to player protos](https://github.com/lambdaclass/mirra_backend/pull/830/commits/7a7651342d1aaf575eb522351da2fc0c737e2efc)
- [Change current_basic_animation if skill is kenzu second or third basic](https://github.com/lambdaclass/mirra_backend/pull/830/commits/aaf3b737ba881a0d3f39315d871caf5d4f77734d)
- [Save player update](https://github.com/lambdaclass/mirra_backend/pull/830/commits/595c257230824e244ea24d24d8b7a46a6b7b3342)
- [Add new current_basic_animation to player complete_entities](https://github.com/lambdaclass/mirra_backend/pull/830/commits/6c6f17c26f59a9d670e139243951e4d79bcb459f)
- [Add new is_combo boolean to messages.proto](https://github.com/lambdaclass/mirra_backend/pull/830/commits/7cc86534f2568a16fdcce882c0797545657c5c54)
- [Differentiating each of Kenzu’s basic combo attacks](https://github.com/lambdaclass/mirra_backend/pull/830/commits/7a47fa870df0865ff3c71aedc4a6765c5a6e4f8b)

## How to test it?

Play a game with game client or unity with the following branch https://github.com/lambdaclass/champions_of_mirra/pull/2014

## Checklist
- [x] Tested the changes locally.
- [x] Reviewed the changes on GitHub, line by line.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: apps/arena/lib/arena/entities.ex [hunk 1]
```diff
@@ -64,7 +64,8 @@ defmodule Arena.Entities do
         on_bush: false,
         bounties: [],
         selected_bounty: nil,
-        bounty_completed: false
+        bounty_completed: false,
+        current_basic_animation: 0
```
Comment (agustinesco) [inline apps/arena/lib/arena/entities.ex:68]: Isn't this value supposed to be 1? to replicate what `get_skill_animation` is doing

File: priv/repo/seeds.exs
Comment (agustinesco) [inline priv/repo/seeds.exs:1]: If this pr leave kenzu usable in the client and it's merged alonside that pr should be activate it in the seeds?

**Review by agustinesco — APPROVED** [review]
LGTM! tested with the client branch in both unity and android and worked like a charm

**Review by AminArria — APPROVED** [review]
LGTM, tested only in Unity

**Pull request merged**